api/proto/attest/attest.proto

```diff
@@ -42,18 +42,44 @@ message ZAttestNonceResp {
   bytes nonce = 1;      //nonce to use in quote generation
 }
 
+//Taken from
+//5.2 Crypto Agile Log Entry Format, from https://trustedcomputinggroup.org/wp-content/uploads/EFI-Protocol-Specification-rev13-160330final.pdf
+message TpmEventLogEntry {
+   uint32 index = 1;           // Sequential index of event in the log
+   uint32 pcr_index = 2;        // PCR index to which this event was measured
+   uint32 event_type = 3;       // The type of this event
+   bytes  digest256 = 4;       // The sha256 digests corresponding to this event
+   bytes  event_data_binary = 5; // The raw event data bytes
+   string event_data_string = 6; // Textual representation of the event data
+}
+
+message GPSCoordinates {
+   double latitude = 1;  // Latitude of the device geo-location
+   double longitude = 2; // Longitude of the device geo-location
```

Comment:
Is there some existing definition/package for lat/long in golang?

Reply:
> Is there some existing definition/package for lat/long in golang?
I could find
https://github.com/kellydunn/golang-geo/blob/master/point.go 
but that is also mostly operating on plain lat/long variables of type float64 (which is equivalent to protobuf's double)

Reply:
OK lets keep these the way you defined them

Reply:
If the device doesn't know (due to no GPS) shouldn't we have a known vs. unknown and/or numeric precision on the reading?
0/0 is a perfectly valid GPS location.